site/index.ts

```diff
@@ -159,10 +161,22 @@ domReady(() => {
             },
             imageUpload: imageUploadOptions,
             externalPlugins: [StackSnippetsPlugin],
-        });
+        };
+
+        const editorInstance = new StacksEditor(place, content.value, options);
+
         // set the instance on the window for developers to poke around in
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
         (window as any)["editorInstance"] = editorInstance;
+
+        if ( place2 != null)
```

Comment:
In JS, you always want to use `===` and the `!==` equivalent. In this case, we can simply update to:

```suggestion
        if (place2)
```